lib/themes/dosomething/paraneue_dosomething/includes/preprocess/preprocess_html.inc

```diff
@@ -25,6 +25,17 @@ function paraneue_dosomething_preprocess_html(&$vars) {
   $member_count = dosomething_user_get_member_count(TRUE);
   $vars['head_title'] = token_replace($vars['head_title'], ['member-count-readable' => $member_count]);
 
+  $enable_facebook_login = variable_get('dosomething_user_social_auth_enabled', FALSE);
+  $vars['enable_facebook_login'] = $enable_facebook_login;
+  if ($enable_facebook_login) {
+    drupal_add_js(PARANEUE_PATH . '/dist/socialAuth.js', [
```

Comment:
This is so nitpicky, but can you make this lowercase dashed? Camel-cased file names for entry points just seems sorta weird to introduce to our naming scheme. 🐫